Add full-text query documentation (#5428)
opensearch-project/documentation-website: _query-dsl/full-text/query-string.md

---
layout: default
title: Query string
parent: Full-text queries
grand_parent: Query DSL
nav_order: 60

redirect_from:
  - /opensearch/query-dsl/full-text/query-string/
  - /query-dsl/query-dsl/full-text/query-string/
---

# Query string query

> A `query_string` query parses the query string based on the [query string syntax](#query-string-syntax). It lets you create powerful yet concise queries that can incorporate wildcards and search multiple fields.

Reviewer: ```suggestion
A `query_string` query parses the query string based on the [query string syntax](#query-string-syntax). It provides for creating powerful yet concise queries that can incorporate wildcards and search multiple fields.
```
